<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelIntersectionClass</class>
 <widget class="QMainWindow" name="ModelIntersectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModelIntersection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,3,1,2" columnstretch="5,2">
    <item row="3" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="intersectButton">
         <property name="maximumSize">
          <size>
           <width>2000</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>求交</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="maximumSize">
          <size>
           <width>20000</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>输出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="gridGroupBox_2">
      <property name="title">
       <string>选择模型</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QPushButton" name="previewButton1">
         <property name="text">
          <string>预览</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="deleteButton1">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>模型1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QComboBox" name="comboBox1"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QComboBox" name="comboBox2"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>模型2</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="previewButton2">
         <property name="text">
          <string>预览</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="deleteButton2">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="horizontalGroupBox">
      <property name="title">
       <string>原始模型</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="TDWidget" name="openGLWidget_2"/>
       </item>
       <item>
        <widget class="TDWidget" name="openGLWidget_3"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="gridGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>载入模型</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="FileButton">
         <property name="text">
          <string>选择模型文件</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="loadButton">
         <property name="text">
          <string>载入</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QGroupBox" name="gridGroupBox">
      <property name="title">
       <string>工作区</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="TDWidget" name="openGLWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>信息</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionFile"/>
    <addaction name="closeAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFile">
   <property name="text">
    <string>选择文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TDWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>tdwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ModelIntersection.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeAction</sender>
   <signal>triggered()</signal>
   <receiver>ModelIntersectionClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
